### PR TITLE
Add get-started.md

### PR DESCRIPTION
**Pull request by swift-kim**
- Add get-started.md
- Rename and re-title configure-device.md

### CODE REVIEW

**Review by bbrto21 — APPROVED** [review]
LGTM

File: doc/get-started.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Getting started
+
+## Setting up the environment
+
+1. Make sure your system and Tizen device meet the minimum requirements in [README](../README.md#system-requirements) and all the required tools are installed.
```
Comment (HakkyuKim) [inline doc/get-started.md:5]: Sequential numbering not explicitly increasing in list

File: README.md [hunk 2]
```diff
@@ -48,32 +48,32 @@ flutter-tizen run
...
-#### Notes
-
-- Only the command line interface is available. We have no plan to add IDE extension support.
+- See [Supported commands](doc/commands.md) for all available commands and their basic usage. See `[command] -h` for more information about each command.
+- See [Getting started](doc/get-started.md) to create your first app and try **hot reload**.
 - To **update** the flutter-tizen tool, run `git pull` in this directory.
 
 ## Docs
 
 #### Tizen basics
```
Comment (HakkyuKim) [inline README.md:57]: Header increased by two levels at a time

**Review by HakkyuKim — COMMENTED** [review]
These are minor issues if the Markdown-HTML renderer understands how to deal with them.
1. Header increased by two levels at a time
2. Sequential numbering not explicitly increasing in list

File: doc/get-started.md (hunk 1 above)
Comment (swift-kim) [reply doc/get-started.md:5]: This is legal.
https://guides.github.com/features/mastering-markdown/

File: README.md (hunk 2 above)
Comment (swift-kim) [reply README.md:57]: I find this is not a problem as `####` is just a synonym of HTML's h4 tag.

**Comment by HakkyuKim** [comment]
> This is legal.
> https://guides.github.com/features/mastering-markdown/

> I find this is not a problem as `####` is just a synonym of HTML's h4 tag.

I thought too much on opening the docs on other web/IDE platforms. For GitHub and most platforms, they're not an issue.

**Review by HakkyuKim — APPROVED** [review]

**Pull request merged**